testing/test_pytest_html.py

```diff
@@ -14,20 +14,15 @@
 pytest_plugins = ("pytester",)
 
 
-def handle_tr_writer_deprecation():
-    # Remove this function when they've fixed
+def remove_deprecation_from_recwarn(recwarn):
+    # TODO: Temporary hack until they fix
     # https://github.com/pytest-dev/pytest/issues/6936
-    import warnings
-    from _pytest.warnings import _setoption
-
-    arg = "ignore:TerminalReporter.writer:pytest.PytestDeprecationWarning"
-    _setoption(warnings, arg)
+    for index, item in enumerate(recwarn):
+        if "TerminalReporter.writer" in repr(item.message):
+            del recwarn.list[index]
```

Comment:
I might do something like this: (sorry for laziness, couldn't get github multiline suggest to work)

```py
def filter_warnings(recwarn):
  # TODO..
  return [w for w in recwarn.list if "TerminalReporter.writer" in repr(w.message)]
```

and then below

```py
warnings = filter_warnings(recwarn)
assert len(warnings) == 0
```

Reply:
Thanks! 🙏